config/chime_science_run_gpu.yaml

```diff
@@ -439,22 +439,22 @@ vis_buffers:
   visbuf_eig_10s_merge:
     kotekan_buffer: vis
   # Buffers for gated
-  visbuf_psr0_5s_0:
+  visbuf_trk0_5s_0:
```

Comment:
I think these are unrelated buffers. In this case `psr` is talking about gated N^2 pulsar data...